src/database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -20,6 +20,7 @@ public function up()
             $table->string('email')->unique();
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
+            //$table->string('api_token', 80)->unique()->nullable();
```

Comment:
if it isn't needed, remove it. Since we're using git, you can find this in the history

Reply:
fixed